src/Workspaces/Remote/ServiceHub/Host/RemoteAnalyzerAssemblyLoaderService.cs

```diff
@@ -1,30 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Composition;
-using System.IO;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.Host.Mef;
-
-namespace Microsoft.CodeAnalysis.Remote.Diagnostics;
-
-/// <summary>
-/// Customizes the path where to store shadow-copies of analyzer assemblies.
-/// </summary>
-[ExportWorkspaceService(typeof(IAnalyzerAssemblyLoaderProvider), [WorkspaceKind.RemoteWorkspace]), Shared]
-[method: ImportingConstructor]
-[method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-internal sealed class RemoteAnalyzerAssemblyLoaderService(
-    [ImportMany] IEnumerable<IAnalyzerAssemblyResolver> externalResolvers)
-    : IAnalyzerAssemblyLoaderProvider
-{
-    private readonly ShadowCopyAnalyzerAssemblyLoader _shadowCopyLoader =
-        new(AbstractAnalyzerAssemblyLoaderProvider.GetDefaultShadowCopyPath(), externalResolvers.ToImmutableArray());
-
-    public IAnalyzerAssemblyLoaderInternal GetShadowCopyLoader()
-        => _shadowCopyLoader;
```

Comment:
we don't need a special REmoteWorkspace impl.  The default impl does this work already properly.